lib/livingstyleguide/templates/colors-example.html.erb

```diff
@@ -0,0 +1,50 @@
+<script>
+  document.addEventListener('DOMContentLoaded', function() {
+    var colorSwatches = document.getElementsByClassName('livingstyleguide--color-swatch');
+    for (var i = 0; i < colorSwatches.length; i++){
+
+      if (colorSwatches[i].innerHTML != "-"){
+        colorSwatches[i].addEventListener('click', function(event) {
+          if(event.altKey){
+            var copyHexColors = window.getComputedStyle(this, ':after').getPropertyValue('content');
+            copyHexColors = copyHexColors.replace(/'/g, "");
+            selectElementRange(createDiv(copyHexColors));
+          }
+          else{
+            selectElementRange(this.children[1]);
+          }
+        });
+      }
+    }
+  });
+
+  function createDiv(copyHexColors){
+    var divCopy = document.createElement('div');
+    divCopy.className = '-lsg-remove';
+    divCopy.setAttribute('style', 'opacity:0; position: absolute; top: -10000px');
+    divCopy.appendChild(document.createTextNode(copyHexColors));
+    document.body.appendChild(divCopy);
+    return divCopy
+  }
+
+  function selectElementRange(copyElementSelect){
+    var range = document.createRange();
+    range.selectNode(copyElementSelect);
+    window.getSelection().removeAllRanges();
+    window.getSelection().addRange(range);
+
+    try {
+      var successful = document.execCommand('copy');
+
+      if (copyElementSelect.classList.contains('-lsg-remove')){
+         document.body.removeChild(copyElementSelect);
+      }
+      var msg = successful ? 'successful' : 'unsuccessful';
+      console.log('Copy color command was ' + msg);
```

Comment:
Actually we should check if the browser is able to copy first and don’t display the option to copy if not.